.github/workflows/ci.yml

```diff
@@ -18,6 +18,25 @@ env:
   AWS_REGION: us-east-1
 
 jobs:
+  linux-compat-use-openssl:
+    runs-on: ubuntu-20.04 # latest
+    strategy:
+      matrix:
+        image:
+          - fedora-34-x64
+          - opensuse-leap
+          - rhel8-x64
+    steps:
+    - name: Install qemu/docker
+      run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+    - name: Build ${{ env.PACKAGE_NAME }}
+      run: |
+        aws s3 cp s3://aws-crt-test-stuff/ci/${{ env.BUILDER_VERSION }}/linux-container-ci.sh ./linux-container-ci.sh && chmod a+x ./linux-container-ci.sh
+        ./linux-container-ci.sh ${{ env.BUILDER_VERSION }} aws-crt-${{ matrix.image }} build -p ${{ env.PACKAGE_NAME }} --cmake-extra=-DUSE_OPENSSL=ON
+
+  # The following containers need to run without -DUSE_OPENSSL because they do not have OpenSSL packages
```

Comment:
Trivial: For clarity, we might want to specify "linux-compat" instead of "following containers" because it appears as though "linux-compiler-compat" and onward run using -DUSE_OPENSSL